### DISCUSSION TITLE
Not seeing windows_exporter mssql stats

### DISCUSSION THREAD

**Discussion by hanserasmus** [netdata/netdata#13630] [Q&A]
Hi everyone

I followed [https://github.com/prometheus-community/windows_exporter](https://github.com/prometheus-community/windows_exporter) installation how to and was able to get the service installed using this command:
```
C:\Windows\system32>msiexec.exe /i C:\Users\hans\Downloads\windows_exporter-0.19.0-amd64.msi ENABLED_COLLECTORS="[defaults],memory,process,mssql"
```
I can see some metrics on my Windows machine on `http://IP:9182/metrics` like: 
```
windows_mssql_collector_duration_seconds{collector="accessmethods",mssql_instance="MSSQLSERVER"} 0.0019197
windows_mssql_collector_duration_seconds{collector="availreplica",mssql_instance="MSSQLSERVER"} 0.0005823
windows_mssql_collector_duration_seconds{collector="bufman",mssql_instance="MSSQLSERVER"} 0
windows_mssql_collector_duration_seconds{collector="databases",mssql_instance="MSSQLSERVER"} 0.0051183
windows_mssql_collector_duration_seconds{collector="dbreplica",mssql_instance="MSSQLSERVER"} 0
windows_mssql_collector_duration_seconds{collector="genstats",mssql_instance="MSSQLSERVER"} 0.0005148
windows_mssql_collector_duration_seconds{collector="locks",mssql_instance="MSSQLSERVER"} 0.0005148
windows_mssql_collector_duration_seconds{collector="memmgr",mssql_instance="MSSQLSERVER"} 0.0005148
windows_mssql_collector_duration_seconds{collector="sqlerrors",mssql_instance="MSSQLSERVER"} 0
windows_mssql_collector_duration_seconds{collector="sqlstats",mssql_instance="MSSQLSERVER"} 0.0005148
windows_mssql_collector_duration_seconds{collector="transactions",mssql_instance="MSSQLSERVER"} 0.0005148
windows_mssql_collector_duration_seconds{collector="waitstats",mssql_instance="MSSQLSERVER"} 0
```

But none of these are available on the netdata dashboard. I would like to end up with mssql stats inside of netdata to help with proper troubleshooting. Could someone please point me to where I am going wrong?

**Comment by ilyam8** [comment]
Hi, @hanserasmus 🙋‍♂️  Unfortunately, the wmi collector doesn't support mssql stats, the list of supported windows_exporter collectors can be found in [readme](https://github.com/netdata/go.d.plugin/tree/master/modules/wmi#windows-machine-monitoring-with-netdata). It is:

- [cpu](https://github.com/prometheus-community/windows_exporter/blob/master/docs/collector.cpu.md)
- [memory](https://github.com/prometheus-community/windows_exporter/blob/master/docs/collector.memory.md)
- [net](https://github.com/prometheus-community/windows_exporter/blob/master/docs/collector.net.md)
- [logical_disk](https://github.com/prometheus-community/windows_exporter/blob/master/docs/collector.logical_disk.md)
- [os](https://github.com/prometheus-community/windows_exporter/blob/master/docs/collector.os.md)
- [system](https://github.com/prometheus-community/windows_exporter/blob/master/docs/collector.system.md)
- [logon](https://github.com/prometheus-community/windows_exporter/blob/master/docs/collector.logon.md)
- [thermalzone](https://github.com/prometheus-community/windows_exporter/blob/master/docs/collector.thermalzone.md)


If you are interested in adding this feature (collecting mssql stats), consider creating a [feature request](https://github.com/netdata/netdata/issues/new/choose).

**Comment by hanserasmus** [comment]
Hi @ilyam8 thank you for the response. So in other words, at this point, no matter what I use I will not see it in netdata? Is that correct? If I want to graph it I would have to contact the prometheus enpoint directly? (or open a feature request). Just want to get my facts straight so I don't waste anyone's time.

**Reply by ilyam8** [reply]
Yes and yes.

**Reply by cakrit** [reply]
Seems to me like the first question kind of contradicts the second though. The Prometheus collector allows direct contact to the prometheus endpoint, so you will see it Netdata. It won't be very pretty, but it will be there. We can certainly do some custom work to make this default output more useful for MSSQL specifically.
